plugins/modules/api_modify.py

```diff
@@ -465,6 +465,26 @@ def polish_entry(entry, path_info, module, for_text):
     for key, field_info in path_info.fields.items():
         if field_info.required and key not in entry:
             module.fail_json(msg='Key "{key}" must be present{for_text}.'.format(key=key, for_text=for_text))
+    required_one_of = path_info.required_one_of or []
+    for require_list in required_one_of:
```

Comment:
This can now be simplified:
```suggestion
    for require_list in path.required_one_of:
```
Same below for `mutually_exclusive`.